addons/account/i18n/vi.po

```diff
@@ -10023,6 +10023,8 @@ msgid ""
 "You cannot change the value of the reconciliation on this account as it "
 "already has some moves"
 msgstr ""
+"Bạn không thể thay đổi giá trị của trường Cho phép Đối soát của tài khoản "
```

Comment:
I am sorry for that. Because my folk already contains Vietnamese translations that completes abt 90% and sometime we do fix translation and other in a single commit.
Is there a way with git to just send a part of changes in a single commit to your repositoty?